features/list_of_posts.feature

```diff
@@ -0,0 +1,19 @@
+Feature: List of Posts
+  As a reader
+  I want to read posts from my favorite blogger
+  So that I can learn something
+
+
+  Scenario: Clicking enter after URL is entered
+    Given I have the blog home page opened
+    When the blog page loads
+    Then I can see the blog post
+    And the blog posts should have a title
+    And the post should show the author name
+   
+  Scenario: The user is on the blog page
```

Comment:
Again, the steps look good but the scenario is not named correctly. Something like `Clicking new post button` would be better.